samcli/commands/pipeline/bootstrap/cli.py

```diff
@@ -38,8 +38,8 @@
     help="Disable interactive prompting for bootstrap parameters, and fail if any required arguments are missing.",
 )
 @click.option(
-    "--stage-name",
-    help="The name of the corresponding pipeline stage. It is used as a suffix for the created resources.",
+    "--environment-name",
```

Comment:
Will this break customers who were using `--stage-name`?

Reply:
We haven't release this feature yet